src/main/java/com/lastovskiy/app/SystemMain.java

```diff
@@ -0,0 +1,49 @@
+package com.lastovskiy.app;
+
+import com.lastovskiy.app.core.Account;
+import com.lastovskiy.app.core.Portfolio;
+
+import com.lastovskiy.app.core.TransactionManager;
+import com.lastovskiy.app.utils.Utils;
+
+import com.lastovskiy.app.validation.FormatValidator;
+import com.lastovskiy.app.validation.ArgumentsValidator;
+import com.lastovskiy.app.validation.TransactionsValidator;
+
+import java.io.IOException;
+
+
+public class SystemMain {
+    public static void main(String[] args) throws IOException {
```

Comment:
Who will catch the "IOException" ? It is the most awful error handling you can do!